### PR TITLE
Human Data: updated training material section

### PR DESCRIPTION
**Pull request by Nazeeefa**
Added link to Tess query, and links to two publications

### CODE REVIEW

File: pages/your_domain/human_data/human_data.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
 ---
-title: Human data
-contributors: [Niclas Jareborg, Nirupama Benis, Ana Portugal Melo, Pinar Alper, Laura Portell Silva, Wolmar Nyberg Åkerström]
+title: Human Data
```
Comment (bedroesb) [inline pages/your_domain/human_data/human_data.md:2]:
```suggestion
title: Human data
```

We do not capitalize titles

**Review by bedroesb — COMMENTED** [review]
Thank you for your contribution! I have a small comment below.

**Review by pinarpink — APPROVED** [review]
Thanks @Nazeeefa  you have definitely improved the text.

**Comment by bedroesb** [comment]
@Nazeeefa I added you git to our contributors list, so you profile pic + github account is linked. Hope you are fine with that :)

**Comment by bedroesb** [comment]
If you want to add more information to your name in the CONTRIBUTORS list, please do so!

**Review by bedroesb — APPROVED** [review]

**Comment by Nazeeefa** [comment]
Thanks @bedroesb, I'll have a look. Could you kindly share the link to contribution list? I can only access following page https://rdmkit.elixir-europe.org/contributors.html

**Comment by bedroesb** [comment]
sure, I added you here at the bottom: https://github.com/elixir-europe/rdmkit/blob/307ab7c38df0a712d618ec731ea2c9d3ffb7e1b0/_data/CONTRIBUTORS.yaml

**Comment by bedroesb** [comment]
This is a better link sorry: https://github.com/Nazeeefa/rdmkit/blob/master/_data/CONTRIBUTORS.yaml

**Pull request merged**